### ISSUE TITLE
Multi-Image injection 

### ISSUE THREAD

**Issue by DiamondGlassDrill** [SUDO-AI-3D/One2345plus#6]
Hey guys, wonderful work.

I was wondering, if I can inject e.g. 4 real images, I shot, from around an object like 45° each and use ur 3D object gen for interpolating in between the images e.g. 4 more missing images, is this something u are working on?

Or is it possible to inject 4 images or more into ur pipeline and switching off the zero123 generation but using the given images of which I would add an angle estimator to have the perspective as prior for ur 3d mesh generator?

Looking forward.

**Comment by Colin97** [comment]
Yes, injecting multi-view images in on our road map. Please stay tuned.

**Comment by DiamondGlassDrill** [comment]
@Colin97 eagerly looking  forward to it. I prepared my endpoints already just awaiting the engine to accelerate  :)

**Comment by DiamondGlassDrill** [comment]
@Colin97 already a feel of a date for the code release? I am wondering, if I could support you somehow!? Let me know if.

**Comment by fefespn** [comment]
Regarding the multi-view and your road map, do we need to support also the camera positions and intrinsics ? or do you have some tricks to avoid this ?

**Comment by alibalapour** [comment]
I have the same question @fefespn and @DiamondGlassDrill, 
@Colin97 do you have any possible date for this option?

BTW your work is great. Thank you for providing this repository.
